docusaurus/docs/ruby/troubleshooting.mdx

```diff
@@ -442,69 +442,75 @@ You have a couple of options:
 
 ## Rake tasks under tests are run more than once in Queue Mode
 
-Make sure the Rake task is loaded once for each test example in the spec file:
+**`knapsack_pro` version 8.1.1 fixed this issue.**
 
 import Tabs from '@theme/Tabs';
```

Comment:
I think imports like that should be at the top as we do in other places: `docusaurus/docs/ruby/rspec.mdx`.

Reply:
As a rule, I was keeping them closer to where they are used.

See for example at the bottom of this file: https://raw.githubusercontent.com/KnapsackPro/docs.knapsackpro.com/refs/heads/main/docusaurus/docs/ruby/queue-mode.mdx

But we can change that in a separate PR if you have a strong opinion.